llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -2955,6 +2955,14 @@ unsigned findFreePredicateReg(BitVector &SavedRegs) {
   return AArch64::NoRegister;
 }
 
+bool isSve2p1OrSme2InStreaming(const AArch64Subtarget &Subtarget) {
```

Comment:
It would be nice to rename this to something like `bool enableMultiVectorSpillFill(..)` and add a command line option that allows users to disable this feature.